src/utils/elements/getElementsForUpsert.ts

```diff
@@ -97,3 +88,19 @@ export type ExpectedInputData = ElementFields;
 
 const notUndefined = <T>(v: T | undefined): v is T =>
   v !== undefined;
+
+// It would be nice to change this, but we can't without a breaking change. Please change this api once you introduce some breaking change.
+const isInternalId = (possibleId: string) =>
+  possibleId.length == 36 && (possibleId.match(/-/g) || []).length === 4;
```

Comment:
nitpick: how about? `posibbleId.match(/-/g)?.length === 4;` seems little bit more readable to me.
also I like the regex for checking the UUID :D but I guess it can be supposed that there will be nothing else than something similar to uuid.